sunpy/coordinates/ephemeris.py

```diff
@@ -29,9 +29,6 @@
 from sunpy.util.decorators import add_common_docstring
 from .frames import HeliographicStonyhurst
 
```

Comment:
I'm not comfortable with this without resolving the discussion in https://github.com/sunpy/sunpy/issues/3650 first.

I'll move this into it's own PR.